index.html

```diff
@@ -0,0 +1,161 @@
+            <p class="tittle">Tonic</p>
+            <p class="words"><b class="canopy">CANOPY</b><img class="img" src="./images/Counter.png" alt="count">Back End Dev<img class="img" src="./images/Counter.png" alt="count">2015</p>  
+            <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div>
+              <button>html</button>
+              <button>css</button>
+              <button>javascript</button>
+            </div><br>
+            <button class="btn-proj">See Project</button>
+           </div>
+        </div>
+        
+        <!--ABOUT SECTION-->
+        <div class="about">
+            <h2 class="header">About <br> Myself</h2>
+            <p class="section">
+               I’m a software developer! I can help you build a product,feature or 
+               website Look through some of my work and experience! If you like what 
+               you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+           <p class="footer">LET CONNECT</p>
+             <ul class="imgs">
+              <li><a href="#"><img src="./images/twitter.svg" alt="twitter handle"></a></li>
+              <li><a href="#"><img src="./images/github.jpg" alt="github handle"></a></li>
+              <li><a href="#"><img src="./images/medium.jpg" alt="medium handle"></a></li>
+              <li><a href="#"><img src="./images/angellist.jpg" alt="angellisthandle"></a></li>
+             </ul>
+              <div>
+                <button class="btn-proj">Get my resume</button>
+              </div>
+              <br>
+               <h5 class="lang">Languages</h5>
+                <div class="ellipse">
+                  <button><img src="./images/Ellipse1.png" alt="ellipse"><span>JavaScript</span></button>
+                  <br>
+                  <button><img src="./images/Ellipse2.png" alt="ellipse"><span>HTML</span></button>
+                  <br>
+                   <button><img src="./images/Ellipse3.png" alt="ellipse"><span>CSS</span></button>
+                </div>
+                <div class="parag">
+                  <h4>Frameworks</h4>
+                  <hr>
+                  <p>Skills</p>
+                  <hr>
+                </div>
+                <div class="contact--form">
+                  <form action="https://formspere.io/f/myyoyyer" method="post">
+                    <h2 class="last--para">Contact me</h2>
+                    <p class="section2">If you have an application you are interested in developing,
+                       a feature that you need built or a project that needs coding.
+                        I’d love to help with it 
+                    </p><br>
+                    <input type="user name" class="name" name"name" placeholder="Sirri Ryisa" required><br><br>
+                    <input type="email" class="email" name"email" placeholder="sirriryisa@gmail.com" min="10" max="30"><br><br>
```

Comment:
- [ ] There is an issue with the `name` attribute because you are missing the `=` between the attribute and the value. I should look like this `name="name"`. Please, fix this issue to keep your code readable and maintainable :pushpin:
- [ ] Also, the correct `type` for the first `<input>` field should be `text` instead of `user name`. You could check a [list of the input types here](https://www.w3schools.com/html/html_form_input_types.asp) :wink:
- [ ] I would also suggest that `name` input should validate the length of the text with a `maxlength="30"` according to the project requirements. You could check this [input validation guide](https://www.w3schools.com/tags/att_input_maxlength.asp)
- [ ] According to the project requirements you should make the `email` input field required, just like you did with the `name` field. Please, kindly address as it is required :hugs: